### PR TITLE
Cassiopeia/adjust toggle google font

### PR DESCRIPTION
**Pull request by hans2103**
Pull Request for Issue # .

### Summary of Changes

Current implementation to toggle the use of Google Fonts differs from the other toggles on tab Advanced of the Cassiopeia template. This PR solves this by adjusting the field parameters. 

### Testing Instructions

- Checkout branch development
- Login to backend.
- Edit the Cassiopeia template style and check the "Advanced" Tab.
- Notice the layout of the _Google Fonts or Text and Heading_ part
- Apply this PR and refresh the page. 
- Notice the layout of the _Google Fonts or Text and Heading_ part

### Expected result

Schermafbeelding 2020-09-07 om 09 08 31

### Actual result

Schermafbeelding 2020-09-07 om 09 08 15

### CODE REVIEW

File: language/en-GB/tpl_cassiopeia.ini [hunk 1]
```diff
@@ -17,8 +17,7 @@ TPL_CASSIOPEIA_STATIC="Static"
 TPL_CASSIOPEIA_STICKY_LABEL="Sticky Header"
 TPL_CASSIOPEIA_TOGGLE="Toggle Navigation"
 TPL_CASSIOPEIA_XML_DESCRIPTION="Continuing the space theme (Solarflare from 1.0, Milkyway from 1.5 and Protostar from 3.0), Cassiopeia is the Joomla 4 site template based on Bootstrap 4."
-TPL_CASSIOPEIA_FONT_LABEL="Google font for text and headings"
-TPL_CASSIOPEIA_FONT_DESC="Load Google font for text and headings (H1, H2, H3, etc)."
-TPL_CASSIOPEIA_FONT_NAME_LABEL="Google Font Name"
+TPL_CASSIOPEIA_FONT_LABEL="Google Fonts or Text and Headings"
```
Comment (brianteeman) [inline language/en-GB/tpl_cassiopeia.ini:20]: Did you mean to change "for" to "or" ?

**Comment by brianteeman** [comment]
Please alpha sort the language strings

**Comment by hans2103** [comment]
Closing this PR since this Switcher will not make it to Joomla4. It is for development purposes only

**Comment by richard67** [comment]
See PR #61 for the removal of the switcher and change of language strings and more.

**Pull request closed without merging**